### PR TITLE
8685 vic intro page content edits 2

### PR DESCRIPTION
**Pull request by rianfowler**
## Not signed in and not id proofed
not signed in and not id proofed

## Signed in but not id proofed
signed in but not id proofed

## Signed in and id proofed
signed in and id proofed

### CODE REVIEW

**Comment by rianfowler** [comment]
This includes the edits mentioned [here](https://github.com/department-of-veterans-affairs/vets-website/pull/7266#issuecomment-365703044).

File: src/js/vic-v2/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ class IntroductionPage extends React.Component {
               <div className="vic-intro-indent">
                 <div><h6>The photo has to follow all the standards listed here. It must:</h6></div>
                 <ul>
-                  <li>Show a full front view of your face and neck, with full visibility of the face and no shadows, <strong>and</strong></li>
+                  <li>Show a full front view of your face and neck (with no hat, head covering, or headphones covering or casting shadows on your hairline or face, <strong>and</strong></li>
```
Comment (jbalboni) [inline src/js/vic-v2/components/IntroductionPage.jsx]: This is missing a closing paren, I think.

**Review by jbalboni — APPROVED** [review]

File: src/js/vic-v2/components/IntroductionPage.jsx (hunk 1 above)
Comment (rianfowler) [reply src/js/vic-v2/components/IntroductionPage.jsx]: Fixed

File: src/js/vic-v2/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ class IntroductionPage extends React.Component {
               <div className="vic-intro-indent">
                 <div><h6>The photo has to follow all the standards listed here. It must:</h6></div>
                 <ul>
-                  <li>Show a full front view of your face and neck, with full visibility of the face and no shadows, <strong>and</strong></li>
+                  <li>Show a full front view of your face and neck (with no hat, head covering, or headphones covering or casting shadows on your hairline or face), <strong>and</strong></li>
```
Comment (rianfowler) [inline src/js/vic-v2/components/IntroductionPage.jsx]: Images show missing closing paren but I fixed it here.

**Comment by rianfowler** [comment]
These changes are ready for review @peggygannon

**Review by peggygannon — CHANGES_REQUESTED** [review]
@rianfowler thanks for missing that missing end paren. 
Can you please also fix the typo for the word after in signed in/not ID proofed
- Subway bullet #2 please fix "Afer" to After

**Comment by rianfowler** [comment]
Typo fixed. @peggygannon 
screen shot 2018-02-19 at 15 30 12

**Review by peggygannon — APPROVED** [review]

**Comment by peggygannon** [comment]
@rianfowler Ok, thanks for fixing the after typo!

**Comment by rianfowler** [comment]
screen shot 2018-02-19 at 15 33 07

**Pull request merged**